src/user/managing-dashboards.md

```diff
@@ -264,28 +227,28 @@ For best print results:
 
 #### Print dashboard layout
 
-Dashboard layout print will approximate the dashboard layout as it is shown in the browser. Note that some adjustments may need to be made to the layout in order to avoid page breaks: the position of some items may be adjusted downwards, and items that are taller than one page are shortened.
+Dashboard layout print will approximate the dashboard layout as it is shown in the browser. Note that some adjustments may need to be made to the layout in order to prevent items being split onto multiple pages: items that would be cut off by a page break are moved to the next page, and items that are taller than one page are shortened to fit on one page.
```

Comment:
```suggestion
*Dashboard layout* print will approximate the dashboard layout as it is shown in the browser. Note that some adjustments may need to be made to the layout in order to prevent items being split onto multiple pages: items that would be cut off by a page break are moved to the next page, and items that are taller than one page are shortened to fit on one page.
```

Reply:
done